server.go

```diff
@@ -32,6 +32,7 @@ func main() {
 	router.HandleFunc("/profile", routes.GetProfileHandler).Methods("GET")
 	router.HandleFunc("/foodtrucks", routes.PostFoodTrucksHandler).Methods("POST")
 	router.HandleFunc("/reviews", routes.PostReviewsHandler).Methods("POST")
+	router.HandleFunc("/users/{userID}", routes.PutUpdateUserHandler).Methods("PUT")
```

Comment:
maybe /user instead of /users since it's the current user? or /profile?

Reply:
ok, and will remove userID field

Reply:
oh wait, how will we know which document to change then without the userID?

Reply:
the auth middleware parses the auth token and puts it as a variable that you can access in every route that goes through the middleware